pgjdbc/src/main/java/org/postgresql/core/PGStream.java

```diff
@@ -47,7 +47,7 @@
   private long nextStreamAvailableCheckTime;
   // This is a workaround for SSL sockets: sslInputStream.available() might return 0
   // so we perform "1ms reads" once in a while
-  private int minStreamAvailableCheckDelay = 1000;
+  private int minStreamAvailableCheckDelay = 1000000;
```

Comment:
How long of delay do we want?
Looks like 1000 ms (1 second) previously. Now it is 1000000 ns, which is 1 ms, right?
I think this is too short.